recipe/recipe.go

```diff
@@ -1,35 +1,38 @@
 package recipe
 
+//List - struct for list of recipes
 type List struct {
-	Count        int        `json:"count"`
-	Start        int        `json:"start_index"`
-	ItemsPerPage int        `json:"items_per_page"`
-	Items        []Response `json:"items"`
-	TotalCount   int        `json:"total_count"`
+	Count        int        `bson:"count,omitempty" json:"count,omitempty"`
+	Start        int        `bson:"start_index,omitempty" json:"start_index,omitempty"`
```

Comment:
I would tend to argue to update swagger spec to change `start` for `offset` and `items_per_page` to `limit` as this follows other API documentation

Reply:
`items_per_page` is more self-explanatory than limit in my opinion. There is ambiguity when naming it to `limit` (such as limit of what)

Reply:
This has always caused fruit for discussion, `items_per_page`, what is page in the context of the data being returned?

limit and offset are common terms when dealing with lists and seen across many restful API's, this model represents a resource with a limited number of items to be returned.

Reply:
I won't force you to change it here, but I think the specification for a list of recipes need to be discussed